compiler/luci/profile/src/CircleNodeOrigin.cpp

```diff
@@ -141,13 +141,22 @@ namespace luci
 
 bool has_origin(const luci::CircleNode *circle_node)
 {
-  return circle_node->annot<CircleNodeOriginAnnotation>() != nullptr;
+  if (circle_node->annot<CircleNodeOriginAnnotation>() == nullptr)
+    return false;
+  if (circle_node->annot<CircleNodeOriginAnnotation>()->origin() == nullptr)
+    return false;
```

Comment:
Maybe errors wouldn't occur without this line, would they?

Reply:
Ah, I added this for another purpose but not yet removed :(
ignore this :)